crates/runtime/src/lib.rs

```diff
@@ -57,6 +65,7 @@ impl Runtime {
             self.config.http_bind_address,
             self.app.clone(),
             self.df.clone(),
+            self.models.clone(),
```

Comment:
We'll need to support dynamic updates (the item Sergei is working on) - which cloning doesn't support. We'll need to rethink how to load data in here. Ok for this PR